libsolidity/codegen/CompilerContext.cpp

```diff
@@ -26,6 +26,8 @@
 #include <libsolidity/ast/AST.h>
 #include <libsolidity/codegen/Compiler.h>
 #include <libsolidity/interface/Version.h>
```

Comment:
need to add <boost/algorithm/string/replace.hpp> to use replace_all_copy in os x.